code/game/objects/items/storage/backpack.dm

```diff
@@ -339,7 +339,7 @@
 
 /obj/item/storage/backpack/marine/tech
 	name = "\improper TGMC technician backpack"
-	desc = "The standard-issue backpack worn by TGMC technicians. Specially equipped to hold sentry gun and M56D emplacement parts."
+	desc = "The standard-issue backpack worn by TGMC technicians. Specially equipped to hold sentry gun and HSG-102 emplacement parts."
```

Comment:
?

Reply:
you see, the M56D is the CM IFF heavy machinegun, we have instead the HSG-102, the heavy smartgun (still called the Tl-102 in code since old name). The engie backpack can fit TL-102 parts, not M56D

![image](https://user-images.githubusercontent.com/40152611/188266832-0531cb1b-e3f6-4851-a7e7-8cf49d14913e.png)